### ISSUE TITLE
Encapsulates the event in json format  Missing required parameter in the JSON body

### ISSUE THREAD

**Issue by havok4u** [noirbizarre/flask-restplus#52]
I was wondering, when the following is my data model that everybody sees
{
  "type": "string",
  "siteId": "string",
}

Then why do I have to encapsulate it with the argument passed in add_argument as follows?
{"event":{
  "type": "VNF",
  "siteId": "DC1",
}}

How would the user possibly know that they are to encapsulate in with "event"?  If I don't I get the following error:

{
  "message": {
    "event": "(Encapsulates the event in json format)  Missing required parameter in the JSON body"
  }
}

Thanks in advance

**Comment by havok4u** [comment]
The "event" is defined in my add_argument
eParser.add_argument('event',type=str, required=True, help = 'Encapsulates the event in json format', location='json')
With this in it wants 'event' as the top of json.  I can put required=False, and then no error message, but my data comes back as null, as it is using the parser I defined to break up the arguments.

**Comment by havok4u** [comment]
Never mind, can close this issue out.  Found the answer to this problem and another.  I was using one parser description for something that had multiple fields, thinking as the location was json, it must all go in one submission.  This does work, but forces you to use the label/name described in the add_argument part of parser as your first key in the dictionary.  I changed it to have multiple fields and where I had lists put type=list in add_argument and it fixed my issue.
